examples/with-account-linking/README.md

```diff
@@ -41,19 +41,16 @@ We are adding a new (`/link`) page where the user can add new login methods to t
 The demo uses the pre-built UI, but you can always build your own UI instead.
 
 -   We do not need any extra configuration to enable account linking
--   To enable manual linking through a custom callback page, we add `getRedirectURL` to the configuration of the social login providers.
 -   We add a custom page (`/link`) that will:
     -   Get and show the login methods belonging to the current user
     -   Show a password form (if available) that calls `/addPassword` to add an email+password login method to the current user.
     -   Show a phone number form (if available) that calls `/addPhoneNumber` to associate a phone number with the current user.
     -   Show an "Add Google account" that start a login process through Google
--   We add a custom page (`/link/tpcallback/:thirdPartyId`) that will:
-    -   Call `/addThirdPartyUser` through a customized `ThirdPartyEmailPassword.thirdPartySignInAndUp` call
 
 ### On the backend
 
 -   We enable account linking by initializing the recipe and providing a `shouldDoAutomaticAccountLinking` implementation
--   We add `/addPassword`, `/addPhoneNumber` and `/addThirdPartyUser` to enable manual linking from the frontend
+-   We add `/addPassword` and`/addPhoneNumber` to enable manual linking from the frontend. Linking thirdparty accounts are handled by the built-in sign-in/up endpoint.
```

Comment:
- can't the add password also be handled by the sign up API?
- can't the add phone number be handled by the passwordless create and consume code API?

Reply:
- Adding the password could be done through the sign up API fairly easily, but then we'd need to get the email address to the frontend and/or do it in 2 requests
- Same here: we could do it on the frontend, but it'd take 2 requests and in general it feels cleaner to implement logic like that on the BE

Reply:
Yea, but asking users to make their custom API feels like more work for the user. So i prefer it to be lesser implementation work, even if it means extra API calls to the backend.